public/v4/apps/baserow.yml

```diff
@@ -0,0 +1,90 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: tiredofit/baserow:$$cap_baserow_tag
+        depends_on:
+            - $$cap_appname-db
+        volumes:
+            - '$$cap_appname-data:/data'
+        environment:
+            ZABBIX_HOSTNAME: $$cap_appname.$$cap_root_domain
+            TIMEZONE: $$cap_timezone
+            API_HOSTNAME: api.$$cap_appname.$$cap_root_domain
+            APPLICATION_HOSTNAME: $$cap_appname.$$cap_root_domain
+            MEDIA_HOSTNAME: media.$$cap_appname.$$cap_root_domain
+            DB_HOST: srv-captain--$$cap_appname-db
+            DB_PORT: 5432
+            DB_USER: $$cap_postgres_user
+            DB_PASS: $$cap_postgres_password
+            DB_NAME: $$cap_postgres_database
+            BACKEND_WORKERS: $$cap_backend_workers
+            PASSWORD_RESET_MAX_AGE: $$cap_password_reset_max_age
+        restart: always
+
+    $$cap_appname-db:
+        image: tiredofit/postgres:$$cap_postgres_tag
```

Comment:
why do not uses postgres/postgres ? (I'm stuck, my server do not successfully pull the image :s)

Reply:
more infos : 
Pulling this repo last almost 10 minutes so the process does not end.
I may retry another day to check if this is a problem with my connexion

Reply:
Use tireofit/postgres to stick with original docker-compose.yml of tireofit/baserow.

Reply:
I am waiting for caprover/caprover/pull/1097 to have more tests. Then, we can try the official Postgres image